test/e2e-go/cli/algoh/expect/algohTimeoutTest.exp

```diff
@@ -57,6 +57,9 @@ if { [catch {
         timeout {::Algoh::Abort "algoh failed to start";}
     }
 
+    # allow algoh time to put files in Node dir
+    ::Algoh::WaitForFile $TEST_NODE_DIR/algod.pid
+
```

Comment:
It's ok to wait for the file, but maybe the reason for this is because the above spawn should have waited until the EOF have reached ?

Reply:
I always thought `spawn` waits.

Reply:
no, spawn definitely doesn't wait. When you use expect, it *could* be waiting if you include an eof section.
also, there is no error code testing here, which is needed. ( i.e. error code testing implies waiting. )

Reply:
I'm ok in merging this, but not sure if the  ::Algoh::WaitForFile is still needed.

Reply:
It is needed, spawn returns earlier than files gets created, and we do have tools to control spawn in a way we did with `goal node start`